pkg/features/pilot/pilot.go

```diff
@@ -113,6 +113,13 @@ var (
 		}
 		return time.Second * time.Duration(duration)
 	}
+
+	// EnableLocalityLoadBalancing provides an option to enable the LocalityLoadBalancerSetting feature
+	// as well as prioritizing the sending of traffic to a local locality. Set the environment variable to any value to enable.
+	// This is an experimental feature.
+	EnableLocalityLoadBalancing = func() bool {
+		return len(os.Getenv("PILOT_ENABLE_LOCALITY_LOAD_BALANCING")) != 0
```

Comment:
can you do a simpler == "1" or something? to remain consistent with other code.. and if existing code is sucky, we could fix them in a separate pass.

Reply:
You mean instead of a function? It has to be a function in order to unit test it.

Reply:
```suggestion
		return os.Getenv("PILOT_ENABLE_LOCALITY_LOAD_BALANCING")) == "1"
```